docs/README.md

```diff
@@ -1,57 +1,44 @@
-
-## Top-level API
-
-enigma.js exposes an API for consuming backend services that are configured with settings.
-
-### `create( config )`
-
-| Property | Type   | Description |
-|----------|--------|-------------|
-| `config` | Object | Service configuration, see docs for details. |
-
-Returns an enigma.js service used to communicate with backend services.
-
-## Usage
-
-enigma.js can be used in CommonJS, AMD, and vanilla browser (global) formats. Install it by running:
-
-```sh
-npm install --save enigma.js
-```
-
-Basic usage is to pull in the library, call `create` and start using the API.
-
-```javascript
-const enigma = require('enigma.js');
-
-enigma.create({ /* Service config */ }).open().then((global) => {
-  // Continue working with the global instance
-}, (err) => {
-  // There was an error getting the global instance
-});
-```
-
-## Services
-
-- [The QIX service](qix/README.md)
+The enigma.js documentation is split into several sections (see below). Unless otherwise noted, all examples assumes a Node.js environment.
+
+- [Getting started](#getting-started)
+- [High-level concepts](./concepts.md#high-level-concepts)
+  - [Node.js versus browser](./concepts.md#nodejs-vs-browser)
+  - [Promises](./concepts.md#promises)
+  - [Authentication](./concepts.md#authentication)
+  - [JSON-RPC protocol](./concepts.md#json-rpc-protocol)
+  - [Generic object model](./concepts.md#generic-object-model)
+- [API documentation](./api.md#api-documentation)
+  - [Configuration](./api.md#configuration)
+    - [Mixins](./api.md#mixins)
+    - [Interceptors](./api.md#interceptors)
+  - [`enigma.create(config)`](./api.md#enigmacreateconfig)
+  - [Session API](./api.md#session-api)
+    - [`session.open()`](./api.md#sessionopen)
+    - [`session.close()`](./api.md#sessionclose)
+    - [`session.suspend()`](./api.md#sessionsuspend)
+    - [`session.resume()`](./api.md#sessionresumeonlyifattachedfalse)
+    - [Event: `opened`](./api.md#event-opened)
+    - [Event: `closed`](./api.md#event-closed)
+    - [Event: `suspended`](./api.md#event-suspended)
+    - [Event: `resumed`](./api.md#event-resumed)
+    - [Event: `notification:<name>`](./api.md#event-notificationname)
+    - [Event: `notification:*`](./api.md#event-notification)
+    - [Event: `traffic:sent`](./api.md#event-trafficsent)
+    - [Event: `traffic:received`](./api.md#event-trafficreceived)
+    - [Event: `traffic:*`](./api.md#event-traffic)
+  - [Generated API](./api.md#generated-api)
+    - [Event: `changed`](./api.md#event-changed)
+    - [Event: `closed`](./api.md#event-closed)
+  - [Sense utilities API](./api.md#sense-utilities-api)
+    - [Configuration](./api.md#configuration-1)
+    - [`SenseUtilities.buildUrl()`](./api.md#senseutilitiesbuildurlconfig)
+- Examples
+  - Sample projects
+    - X
```

Comment:
I am not sure yet if it should be in here, or in the root README.md. This file is already pretty large so I might move this out?